aireview/src/aireview/domain/entities/review_comment.py

```diff
@@ -1,8 +1,19 @@
 from dataclasses import dataclass
+from typing import Any
+
 
 @dataclass
 class ReviewComment:
     file_path: str
     line: int
```

Comment:
**style**: Default values ('', 0) in get() calls could mask data validation issues. Consider raising ValueError for missing required fields instead of using defaults.